module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -700,19 +729,21 @@ protected function getHoldingDetailsForItem($holding): array
      * location (in the case where no items are attached to a holding) or the item record
      * location (in cases where there are attached items).
      *
-     * @param string $locationId     Location identifier from FOLIO
-     * @param array  $holdingDetails Holding details produced by getHoldingDetailsForItem()
+     * @param string     $locationId     Location identifier from FOLIO
+     * @param array      $holdingDetails Holding details produced by getHoldingDetailsForItem()
+     * @param ?\stdClass $currentLoan    Any current loan on this item
      *
      * @return array
      */
     protected function getItemFieldsFromLocAndHolding(
         string $locationId,
         array $holdingDetails,
+        ?\stdClass $currentLoan = null,
```

Comment:
Does this mean the function name needs to change?  FYI @dltj 

Or if we really wanted to keep the function scope to just things determinable from the location and holding, then we could calculate `is_holding` outside of the function intstead.

Reply:
I have no strong feelings about whether to separate the is_holding logic, or rename the function to something like getItemFieldsFromLocHoldingAndLoan... but I do think it would be a good idea to do one of those two things.

Reply:
Yeah agreed.  @dltj what do you prefer?

Reply:
I think I'm a bigger fan of fewer class methods even though a name like `getItemFieldsFromLocHoldingAndLoan()` is getting somewhat unwieldy. I am > < ("this close") to plugging the whole thing into an LLM and asking for advice on a briefer but just as descriptive method name.

Reply:
A few thoughts:

1.) It looks to me like the comment on this method is inaccurate, so whatever we call the method, we should fix the comments. (I think the comments pre-date the separation of item / non-item data).

2.) We could use a simpler name like `getBaseItemFields` since the purpose of this method is to get the "common-denominator" fields that apply in all scenarios. Similarly, something like `getCommonItemFields` or `getUniversalItemFields` might convey the same idea. The comment can elaborate on what it means.

3.) Or, if the point of this is to get everything except the item-level fields, we could use something like `getItemFieldsFromNonItemData`.

Is that any help?

Reply:
Huh, yeah, `getItemFieldsFromNonItemData` might be the most descriptive with a method comment about what that means. Does that work with what you envision, Maccabee?

Reply:
Thanks both.  I went with @dltj 's suggestion, I think it encapsulates what we're doing best.

Reply:
Works for me. I just pushed a small commit to fix a single-letter typo in the comment. :-)